api/src/main/java/org/eclipse/microprofile/openapi/annotations/PathItem.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.openapi.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.eclipse.microprofile.openapi.annotations.extensions.Extension;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.servers.Server;
+
+/**
+ * Describes a set of operations available on a single path.
+ *
+ * @since 4.0
+ */
+@Target({})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface PathItem {
+
+    /**
+     * The name of the Path Item object, used as the map key when the path item is used under
+     * {@link Components#pathItems()} or {@link OpenAPIDefinition#webhooks()}
+     *
+     * @return the path item name
+     */
+    String name();
+
+    /**
+     * Reference value to a PathItem object.
+     * <p>
+     * This property provides a reference to an object defined elsewhere. This property and properties other than
+     * {@link #name()} are mutually exclusive. If properties other than {@code name} are defined in addition to the
+     * {@code ref} property then the result is undefined.
```

Comment:
Mutual exclusivity between `$ref` and other properties isn't the case for path items, right?

Reply:
Ah, good spot. Will need a test for this too.